kms/snippets.go

```diff
@@ -209,6 +209,7 @@ func getRingPolicy(keyRingName string) (*iam.Policy, error) {
 
 // [END kms_get_keyring_policy]
 
+// [START kms_get_cryptokey_policy]
```

Comment:
Please add newlines between the region tag and the start of the function comments. Otherwise, they can show up in the GoDoc for the function.

Reply:
Done